package.json

```diff
@@ -2,10 +2,21 @@
   "name": "super-repo",
   "version": "2.0.4",
   "description": "Repository-ish pattern for your data, that implements best practices for working with and storing data on the client-side.",
-  "main": "src/index.js",
+  "main": "lib/index.js",
   "scripts": {
+    "build": "babel src --out-dir lib",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "babel": {
+    "presets": [
+      [
+        "es2015",
+        {
+          "modules": false
+        }
+      ]
+    ]
+  },
```

Comment:
I had an issue with [transpiling 'this' to 'undefined'](https://stackoverflow.com/q/34973442/1333836). This is how I fixed it, I am not sure if this is a best practice, but it seems like it works.